ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```
@@ -353,6 +353,21 @@ objects:
         sshKey: |
           ${SSH_PUB_KEY}
         EOF
+            export GOVC_URL=vcsa-ci.vmware.devcluster.openshift.com
+            export GOVC_USERNAME="${VSPHERE_USER}"
+            export GOVC_PASSWORD="${VSPHERE_PASSWORD}"
+            export GOVC_INSECURE=1
+            export GOVC_DATACENTER=dc1
+            export GOVC_DATASTORE=nvme-ds1
+            export OVA_URL="$(jq -r '.baseURI + .images["vmware"].path' /var/lib/openshift-install/rhcos.json)"
+            export VM_TEMPLATE="rhcos-$(echo ${OVA_URL} | sha1sum | cut -b -8)"
```

Hashing was probably not a good idea.
```suggestion
            export VM_TEMPLATE="$(echo ${OVA_URL} | rev | cut -d '/' -f 1 | rev)"
```
This would keep the filename intact, so that these could be cleaned later on

---

b0868d37...77743c1a